src/pages/EditMonitor.vue

```diff
@@ -899,6 +902,28 @@
                                     </div>
                                 </template>
                             </template>
+
+                            <!-- TLS Options -->
+                            <template v-if="monitor.type === 'port-tls'">
+                                <h2 class="mt-5 mb-2">{{ $t("TLS Options") }}</h2>
+
+                                <!-- Use STARTTLS -->
+                                <div class="my-3 form-check">
+                                    <input id="start-tls" v-model="monitor.startTls" class="form-check-input" type="checkbox">
+                                    <label class="form-check-label" for="start-tls">
+                                        {{ $t("Use STARTTLS") }}
+                                    </label>
+                                </div>
+
+                                <!-- Request -->
```

Comment:
This being here means that this is part of the TLS-options. Is this intentional?

Reply:
I hesitated to give the request too prominent a location in the _General_ section. For one because it's an optional field, for another because it's a multi-line string and I was afraid of cluttering up the main column. But you're right, it has less to do with TLS than with the monitor itself. Also, this is another field that could easily apply to the existing "TCP (Port)" monitor.

I can think of some options we have here:

1. Move it into the _General_ section.
2. Move it into the _General_ section together with the "Use STARTTLS" checkbox since the latter would be the only element left in the _TLS Options_. (I can imagine there being more in the future, though, e.g. "Warn about soon-to-expire certificate", which was requested [here](https://github.com/louislam/uptime-kuma/pull/1626#issuecomment-1425685769).)
3. Rename _TLS Options_ to something like _Optional_ (better name?) to avoid cluttering up the _General_ section.

You may have a better feel than I do of what's most natural for the Uptime Kuma UI. What do you think?

Reply:
This will get a redesign anyway after merging with the "TCP (Port)" monitor and adding more TLS-specific options.